macos/FluentUI/Button/Button.swift

```diff
@@ -5,158 +5,254 @@
 
 import AppKit
 
+// MARK: - Button Formatting Parameters
+
+/// Indicates the size of the button
+@objc(MSFButtonSize)
+public enum ButtonSize: Int, CaseIterable {
+	case large
+	case small
+}
```

Comment:
Somewhat related, we offhand mentioned standardizing the "size" type in FluentUI React Native, which I put in this bug here https://github.com/microsoft/fluentui-react-native/issues/475

Doesn't affect your change, but is great we are using the same names!

Reply:
RIP `medium`, you were so much help figuring out the alignment algorithm or lack thereof, before you were deemed redundant.

Reply:
alignment algorithm? I guess it is redundant so I don't need to know :l